client/components/jetpack-product-card/features.tsx

```diff
@@ -0,0 +1,69 @@
+/**
+ * External dependencies
+ */
+import { useTranslate } from 'i18n-calypso';
+import { isArray, isObject } from 'lodash';
+import React, { useState, useCallback, FunctionComponent } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import ExternalLink from 'components/external-link';
+import FoldableCard from 'components/foldable-card';
+import FeaturesItem from './features-item';
+import { Features } from './types';
+
+export type Props = {
+	features: Features;
+	isExpanded?: boolean;
+};
+
+const JetpackProductCardFeatures: FunctionComponent< Props > = ( {
+	features,
+	isExpanded: isExpandedByDefault,
+} ) => {
+	const [ isExpanded, setExpanded ] = useState( !! isExpandedByDefault );
+	const onOpen = useCallback( () => setExpanded( true ), [ setExpanded ] );
+	const onClose = useCallback( () => setExpanded( false ), [ setExpanded ] );
```

Comment:
Should we track these events? If so, we could use the `useTrackCallback` hook from `lib/jetpack/use-track-callback`.

Reply:
It could be a good thing indeed. I'm adding that.